msg/sensor_accel_fifo.msg

```diff
@@ -4,7 +4,9 @@ uint64 timestamp_sample
 uint32 device_id          # unique device ID for the sensor that does not change between power cycles
 
 float32 dt                # delta time between samples (microseconds)
-float32 scale
+float32 scale             # sensor resolution scale (LSB/g)
+
+uint16 dynamic_range      # dynamic range of the sensor (g)
```

Comment:
We could do a quick check, but off the top of my head I'm not aware of any cases where the range isn't already limited by the int16 min/max, so we already have the range from the scale.